lib/signs_ui_web/router.ex

```diff
@@ -29,9 +29,12 @@ defmodule SignsUiWeb.Router do
     get "/", PageController, :index
     get "/signs", SignsController, :index
     post "/signs", SignsController, :update
+    get "/messages", MessagesController, :index
   end
 
   scope "/", SignsUiWeb do
+    pipe_through [:api]
+    post "/messages/update", MessagesController, :update
```

Comment:
The conventional endpoint here would probably be `post "/messages`, MessageController, :create`

Reply:
Also, I don't think we should mess with the `/_health` endpoint. It doesn't accept JSON, for example. You should probably put this in its own scope.

Reply:
thats true, originally i had this as updating a list of messages, so i should probably make this change to reflect what it really is now.